src/ripple/consensus/LedgerTrie.h

```diff
@@ -395,6 +395,22 @@ class LedgerTrie
         return std::make_pair(curr, pos);
     }
 
+    Node*
```

Comment:
Nit: Consider adding documentation for this function. Its internal, but might be helpful nonetheless.